lib/LedgerSMB/Report/Orders.pm

```diff
@@ -15,6 +15,7 @@ use namespace::autoclean;
 extends 'LedgerSMB::Report';
 with 'LedgerSMB::Report::Dates';
 use LedgerSMB::MooseTypes;
+use LedgerSMB::Magic qw( OEC_SALES_ORDER OEC_PURCHASE_ORDER OEC_QUOTATION OEC_RFQ );
 
 =head1 DESCRIPTION
 
```

Comment:
@ziuq you can safely ignore this exploratory comment. @ehuelsmann and I will discuss later, but it may not be worth making a change here.